spec/models/green_flag/feature_spec.rb

```diff
@@ -132,4 +132,55 @@
       end
     end
   end
+
+  describe "#requires_manual_deletion?" do
+    let(:feature)           { GreenFlag::Feature.create(code: 'big_font') }
+    let(:feature_decisions) { [] }
+
+    context "with more than 1,000,000 feature decisions" do
+      it 'returns true' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.